.changeset/angry-comics-pay.md

```diff
@@ -0,0 +1,15 @@
+---
+'livepeer': minor
+---
+
+**Feature:** Asset Metrics
+
+There is now support for asset metrics, with total view count being returned
+when metrics have been reported to the correct reporting URL (this is handled in
+`@livepeer/react`'s `VideoPlayer`).
+
+```typescript
+const metrics = getMetrics({ assetId });
```

Comment:
Should this be `getAssetMetrics` as well?

Reply:
Actually, seems like this file is outdated? Maybe the right changeset is the beige-rockets-notice below. Or is this one on the core and the other one on react?